pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   formz: ^0.4.0
   intl: ^0.17.0
   uuid: ^3.0.5
+  flutter_slidable: any
```

Comment:
As I said, I slightly prefer the long-press, which also doesn't require another library, but this one seems well maintained so it's probably okay to go ahead with this now.